concourse/concourse.yml

```diff
@@ -0,0 +1,199 @@
+---
+name: concourse
+
+# Meta property overrides
+meta:
+  az: ~
+  atc:
+    db:
+      name: ~
+      user: ~
+      pass: ~
+    basic_auth:
+      user: ~
+      pass: ~
+  network:
+    range: ~
+    reserved: ~
+    dns: ~
+    gateway: ~
+    discovery_ip: ~
+    subnet: ~
+    security_groups: ~
+  disk:
+    database:
+      size: ~
+      type: ~
+  resources:
+    web:
+      instance_type: ~
+      elbs: ~
+    discovery:
+      instance_type: ~
+    databases:
+      instance_type: ~
+    workers:
+      instance_type: ~
+  compilation:
+    instance_type: ~
+    disk:
+      size: ~
+      type: ~
+
+# replace with bosh status --uuid
+director_uuid: (( merge ))
+
+# pick a sane db name and password
+atc_db_name: &atc-db-name (( meta.atc.db.name ))
+atc_db_role: &atc-db-role
+  name: (( meta.atc.db.user ))
+  password: (( meta.atc.db.pass ))
+
+# replace with an AZ from your account
+availability_zone: &az (( meta.az ))
+
+# replace all addresses with your VPC range
+#
+# e.g. X.X.0.2 -> 10.0.0.2
+networks:
+  - name: concourse
+    type: manual
```

Comment:
We might be able to.  I have not experimented with the dynamic network configuration yet though.  I can test it out in my personal AWS environment and report my results back.